test/fixtures/shared/variables.tf

```diff
@@ -43,3 +43,9 @@ variable "members" {
   description = "An allowed list of members (users, service accounts). The signed-in identity originating the request must be a part of one of the provided members. If not specified, a request may come from any user (logged in/not logged in, etc.). Formats: user:{emailid}, serviceAccount:{emailid}"
   type        = list(string)
 }
+
+variable "random_suffix" {
```

Comment:
There is 0 reason for this to be a variable.

@paulpalamarchuk Just make this a random hex like we do everywhere else.

Reply:
Moved inside `simple_example` fixture